### PR TITLE
Add haml gem

### PR DESCRIPTION
**Pull request by alghazu**
### What we did:
- Add haml gem
- Converted views to ham
- Add files to .gitignore

### CODE REVIEW

File: app/views/layouts/mailer.html.haml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+%html
+  %head
+    %meta{:content => "text/html; charset=utf-8", "http-equiv" => "Content-Type"}/
```
Comment (tochman) [inline app/views/layouts/mailer.html.haml]: Old hash syntax. Please change

**Review by tochman — CHANGES_REQUESTED** [review]
Just one detail. Good job

File: app/views/layouts/mailer.html.haml (hunk 1 above)
Comment (alghazu) [reply app/views/layouts/mailer.html.haml]: Done

**Review by GeorgeJLewis — APPROVED** [review]
LGTM

**Pull request merged**